### PR TITLE
Add AWS Infrastructure

### LINKED ISSUE #3: Push code to AWS Batch

### LINKED ISSUE #9: Move from Jack's personal AWS account to OCF AWS!

### LINKED ISSUE #10: Consider running the EC2 job continuously
2 advantages:

* Don't need SQS (so could save a few quid there, maybe)  (See #8 for reasons why)
* Can process NWPs as soon as they're published (rather than waiting up to an hour)

### PR DESCRIPTION
**Pull request by flowirtz**
This PR adds all relevant AWS infrastructure to run the `metoffice_ec2` script. Everything is provided as Terraform files.

infra

We are subscribing to two SNS Topics from the Metoffice, filtering the messages based on a custom filter policy and send the resulting messages to SQS. To consume those messages we then set up a simple ECS Fargate cluster and run our docker image on a small instance in there.

The resulting files are pushed to S3.

Fixes #10, Fixes #9, Fixes #3

### CODE REVIEW

**Comment by flowirtz** [comment]
(ignore the tf check, I already ran it and fixed it, just can't retrigger it without pushing...)

File: scripts/ec2.py [hunk 1]
```diff
@@ -78,13 +84,17 @@ def main():
         sqs_reply = sqs.receive_message(
```
Comment (JackKelly) [inline scripts/ec2.py:85]: When there's no messages, this `while` loop will be a very tight loop, and so might max out the CPU (maybe?!)  Should we add `WaitTimeSeconds=60` to force `sqs.receive_message` to wait up to 60 seconds if the queue is empty?  [Docs here](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/sqs.html#SQS.Client.receive_message).

```suggestion
        sqs_reply = sqs.receive_message(
            WaitTimeSeconds=60,
```

File: scripts/ec2.py [hunk 2]
```diff
@@ -4,17 +4,23 @@
...
-SQS_URL = 'https://sqs.eu-west-1.amazonaws.com/741607616921/uk-metoffice-nwp'
+SQS_URL_DEFAULT = 'https://sqs.eu-west-1.amazonaws.com/741607616921/uk-metoffice-nwp'
+SQS_URL = os.getenv('SQS_URL', SQS_URL_DEFAULT)
 
-REGION = 'eu-west-1'
+DEST_BUCKET_DEFAULT = 'uk-metoffice-nwp'
+DEST_BUCKET = os.getenv('DEST_BUCKET', SQS_URL_DEFAULT)
 
-DEST_BUCKET = 'uk-metoffice-nwp'
+REGION = 'eu-west-1'
```
Comment (flowirtz) [inline scripts/ec2.py:18]:
```suggestion

# Remember to update infrastructure/inputs.tf as well, when modifying this
```

File: scripts/ec2.py (hunk 1 above)
Comment (flowirtz) [reply scripts/ec2.py:85]: great point!

**Pull request merged**